starters/express-apollo-prisma/README.md

```diff
@@ -0,0 +1,190 @@
+
+### Manual
+
+```bash
+git clone https://github.com/thisdot/starter.dev.git
+```
+
+- Copy and rename the `starters/express-apollo-prisma` directory to the name of your new project.
+- Make sure you have docker & docker-compose installed on your machine
+- `cd` into your project directory and run `npm install`.
+- Make sure you have docker & docker-compose installed on your machine
+- Create a `.env` file and copy the contents of `.env.example` into it.
+- Run `npm run docker:mount` to start the database and the Redis instances
+- Run `npm run start` to start the development server.
+- Open your browser to `http://localhost:4001` to see the API documentation with the existing endpoints.
+
+## Commands
+
+- `npm run docker:mount` - Starts up a Mysql database and two Redis instances for caching
+- `npm run docker:unmount` - Stops the running database and Redis docker containers.
+<!-- TODO - `npm run db:seed` - Allows you to seed the database (See the Seeding section) -->
+- `npm run dev` - Starts the development server (Needs a running infrastructure first)
+- `npm run build` - Builds the app.
+- `npm start` - Starts the built app. (Needs a running infrastructure first)
+- `npm test` - Runs the unit tests.
+- `npm run lint` - Runs ESLint on the project.
+- `npm run format` - Formats code for the entire project.
+- `npm prisma:format` - Updates your database using migrations during development and creates the database if it does not exist.
+- `npm prisma:migrate:reset` - Deletes and recreates the database, or performs a 'soft reset' by removing all data, tables, indexes, and other artifacts.
+- `npm prisma:migrate:dev` - updates your database using migrations during development and creates the database if it does not exist.
+- `npm run prisma:generate` - Generates the API schema types into the `src/interfaces/schema.ts` file
+- `npm run prisma:deploy` - Applies all pending migrations, and creates the database if it does not exist. Primarily used in non-development environments.
+- `npm run docker:build` - Builds, (re)creates, starts, and attaches to containers to a service in the background.
+- `npm run docker:start` - Runs an existing docker container.
+- `npm run docker:stop` - Stops an existing docker container.
+- `npm run docker:down` - Stop and removes the docker container & network.
+- `npm run docker:remove` - Removes stopped docker service containers.
+
+### Database and Redis 
+To start up your API in dev mode with an active database connection, please follow the following steps:
+
+1. create a `.env` file. For the defaults, copy the contents of the `.env.example` file's content into it.
+2. run `npm run docker:mount`
+3. run `npm run dev`
+
+The above steps will make sure your API connects to the database and Redis instances that get started up with docker. When you finish work, run `npm run infrastructure:stop` to stop your database and Redis containers.
+
+### Seeding
+<!-- TODO -->
+
+### Reset infrastructure
```

Comment:
This chapter could use instructions on bringing the infrastructure down. If you simply rerun the command to start the infrastructure, it will keep the existing data and not reset (as the title suggests).

Depending on the results of #916, we might also need to remove cache files.